tests/optimization/tranquilo/test_adjust_radius.py

```diff
@@ -70,7 +70,7 @@ def test_min_radius_is_not_violated(options):
         options=options,
     )
 
-    expected = 1e-6
+    expected = 7.5e-7
```

Comment:
This does not test that min radius is not violated anymore because the min radius is not binding. We need to start with a smaller radius or not use default options to have a meaningful test.